packs/equipment-effects/effect-deadweight-mutagen-greater.json

```
@@ -0,0 +1,73 @@
+{
+    "_id": "IFHqrbQUOT2LzNDB",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Deadweight Mutagen (Greater)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Deadweight Mutagen (Greater)]</p>\n<p><strong>Benefit</strong> You gain a +3 item bonus to Athletics checks to @UUID[Compendium.pf2e.actionspf2e.Item.Shove] and @UUID[Compendium.pf2e.actionspf2e.Item.Trip], to your Fortitude and Reflex DCs against attempts to Shove or Trip you, and to saving throws against effects that attempt to force you to move or knock you @UUID[Compendium.pf2e.conditionitems.Item.Prone]. You can attempt to Shove or Trip creatures up to two sizes larger than you.</p>\n<p><strong>Drawback</strong> You gain the @UUID[Compendium.pf2e.conditionitems.Item.Encumbered] condition and can't remove it while under the effects of the mutagen.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 11
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "action:shove",
+                            "action:trip"
+                        ]
+                    }
+                ],
+                "selector": "athletics",
+                "type": "item",
+                "value": 3
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "action:shove",
+                            "action:trip"
```

```suggestion
                            "action:trip",
                            "inflicts:prone",
                            "inflicts:forced-movement"
```
Not sure on the inflicts on the forced movement but those are mostly so we can easily find these things in compendium later.

---

Good call.